examples/howto/optimize_evoked.py

```diff
@@ -69,15 +69,15 @@
 from hnn_core.optimization import optimize_evoked
 
 with MPIBackend(n_procs=n_procs):
-    params_optim = optimize_evoked(params, exp_dpl, initial_dpl,
-                                   scale_factor=scale_factor,
-                                   smooth_window_len=smooth_window_len)
+    net_opt = optimize_evoked(net.copy(), tstop=tstop, n_trials=1,
```

Comment:
Note that the user can now specify the number of trials that are run and averaged across for a given simulation.

Reply:
nice! could the `net.copy` be done inside the `optimize_evoked` function so it's not visible to the user?